src/app/app.component.ts

```diff
@@ -36,6 +36,14 @@ export class AppComponent {
     },
   ];
 
+  ngxOptions: NgxStoriesOptions = {
```

Comment:
Here as well, because we've already declared `storyOptions`

Reply:
Updated the PR with required changes.